app/assets/javascripts/govuk_publishing_components/vendor/lux/lux-reporter.js

```diff
@@ -1492,13 +1522,36 @@
       logger.logEvent(LogEvent.PaintTimingNotSupported);
       return undefined;
     }
-    function getINP() {
+    function getINPDetails() {
       if (!("PerformanceEventTiming" in self)) {
         return undefined;
       }
-      return getHighPercentileINP();
+      return getHighPercentileInteraction();
+    }
+    /**
+    * Build the query string for the INP parameters:
+    *
+    * - INP: The duration of the P98 interaction
+    * - INPs: The selector of the P98 interaction element
+    * - INPt: The timestamp of the P98 interaction start time
+    * - INPi: The input delay subpart of the P98 interaction
+    * - INPp: The processing time subpart of the P98 interaction
+    * - INPd: The presentation delay subpart of the P98 interaction
+    */
+    function getINPString(details) {
+      return [
+        "&INP=" + details.duration,
+        details.selector ? "&INPs=" + encodeURIComponent(details.selector) : "",
+        "&INPt=" + clamp(floor(details.startTime)),
+        "&INPi=" + clamp(floor(details.processingStart - details.startTime)),
+        "&INPp=" + clamp(floor(details.processingEnd - details.processingStart)),
+        "&INPd=" + clamp(floor(details.startTime + details.duration - details.processingEnd)),
+      ].join("");
     }
     function getCustomerId() {
```

Comment:
I thought this function needs to be as follows, but I can't reproduce the error that it solves. Take a look at this related PR: https://github.com/alphagov/govuk_publishing_components/pull/3592

It's possible the error only happens in production, but I think this code change makes sense regardless.

```JS
function getCustomerId() {
  return LUX.customerid || "";
}
```

I probably need to clarify the documentation on this, apologies.

Reply:
Thanks Andy. Good spot, and I've now reverted the `getCustomerId` function change.